cmd/cfssl/cfssl_test.go

```diff
@@ -1 +1,350 @@
 package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"reflect"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/cloudflare/cfssl/csr"
+	"github.com/cloudflare/cfssl/helpers/testsuite"
+)
+
+// CFSSL local server to start up
+var server *testsuite.CFSSLServer
+
+// We will test on this address and port.
+// Be sure that these are free or the test will fail.
+var addressToTest = "127.0.0.1"
+var portToTest = 8888
+var postStr = fmt.Sprintf("http://%s:%s", addressToTest, strconv.Itoa(portToTest))
+var hosts = []byte(`{"hosts":["www.example.com"],
+					   "names":[{"C":"US", "ST":"California", "L":"San Francisco", "O":"example.com"}],
+					   "CN": "www.example.com"}`)
+var requestStr = []byte(`{ "request": {"hosts":["www.example.com"], 
+						"names":[{"C":"US", "ST":"California", "L":"San Francisco", "O":"example.com"}], 
+						"CN": "www.example.com"} }`)
+var certRequest = []byte(`{"certificate_request": "-----BEGIN CERTIFICATE REQUEST-----\nMIIBUjCB+QIBADBqMQswCQYDVQQGEwJVUzEUMBIGA1UEChMLZXhhbXBsZS5jb20x\nFjAUBgNVBAcTDVNhbiBGcmFuY2lzY28xEzARBgNVBAgTCkNhbGlmb3JuaWExGDAW\nBgNVBAMTD3d3dy5leGFtcGxlLmNvbTBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IA\nBK/CtZaQ4VliKE+DLIVGLwtSxJgtUKRzGvN1EwI3HRgKDQ3l3urBIzHtUcdMq6HZ\nb8jX0O9fXYUOf4XWggrLk1agLTArBgkqhkiG9w0BCQ4xHjAcMBoGA1UdEQQTMBGC\nD3d3dy5leGFtcGxlLmNvbTAKBggqhkjOPQQDAgNIADBFAiAcvfhXnsLtzep2sKSa\n36W7G9PRbHh8zVGlw3Hph8jR1QIhAKfrgplKwXcUctU5grjQ8KXkJV8RxQUo5KKs\ngFnXYtkb\n-----END CERTIFICATE REQUEST-----\n"}`)
```

Comment:
I wonder if these long strings could be moved to testdata files and their contents referenced by filename as in https://github.com/cloudflare/cfssl/blob/master/helpers/helpers_test.go#L16